content/docs/04-clusters/02-data-center/01-maas/15-create-manage-maas-clusters.md

```diff
@@ -0,0 +1,165 @@
+
+# Deploy a MAAS cluster
+
+To deploy a new MAAS cluster:
+
+<br />
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Navigate to the **Main Menu** and click **Clusters**. Then click the **Add New Cluster** button.
+
+
+3. Click **Deploy New Cluster** on the Create a New Cluster page.
+
+
+4. Select **MAAS** and click the **Start MAAS Configuration** button.
+
+
+5. Provide basic cluster information: **Cluster name**, **Description**, and **Tags**.
+
+
+6. Select your MAAS cloud account from the **drop-down Menu** and click **Next**.
+
+
+7. Select the cluster profile for your MAAS cluster. 
+
+
+8. Review and override pack parameters as desired and click **Next**. By default, parameters for all packs are set with values defined in the cluster profile.
+
+
+9. Select a domain from the **Domain drop-down Menu** and click **Next**. 
+
+
+10. Configure the master and worker node pools. Enter the values for properties listed in the following tables.
+
+<br />
+
+
+#### Master Node Pool
+
+| Property | Description |
+|-----------|-------------|
+| Node pool name | A descriptive name for the node pool. |
+| Number of nodes in the pool | Number of nodes to be provisioned for the node pool. For the master pool, this number can be 1, 3, or 5. |
+| Allow worker capability | Select this option to allow workloads to be provisioned on master nodes. |
+| Additional [labels](/clusters/cluster-management/taints#overviewonlabels) | Labels apply placement constraints on a pod. For example, you can add a label to make a node eligible to receive the workload. |
+| [Taints](/clusters/cluster-management/taints#overviewontaints) | Sets toleration to pods and allows (but does not require) the pods to schedule onto nodes with matching taints. |
+| Cloud configuration | Set the Resource Pool and Availability Zone in MAAS to select available servers from for deployment. Filter available servers to only those that have at least the amount of CPU and Memory selected. |
+| Resource pool | The MAAS resource pool from which to select available servers deployment.  |
+| Availability Zones | The MAAS Availability Zones from which to select available servers for deployment. If you select multiple zones, Palette will deploy servers evenly across them as long as sufficient servers are available to do so. |
+
+<br />
+
+
+#### Worker Node Pool
+
+| Property | Description |
+|-----------|-------------|
+| Node pool name | A descriptive name for the node pool. |
+| Number of nodes in the pool | Number of nodes to be provisioned for the node pool.|
+| Additional [labels](/clusters/cluster-management/taints#overviewonlabels) | Optional labels apply placement constraints on a pod. For example, you can add a label to make a node eligible to receive the workload. |
+| [Taints](/clusters/cluster-management/taints#overviewontaints) | Sets toleration to pods and allows (but does not require) the pods to schedule onto nodes with matching taints. |
```

Comment:
@ritawatson both these sections are documented in the cluster management [Node Pools page](https://docs.spectrocloud.com/clusters/cluster-management/node-pool).  Your call, but in the other providers, I have added verbiage and linked to this page.

Reply:
@karl-cardenas-coding I didn't realize that. I'll check verbiage in docs for other providers and refer.

Reply:
Removed tables and added link to Node Pool page.